modules/components/analytics/core/org.wso2.carbon.pc.analytics.core/src/main/java/org/wso2/carbon/pc/analytics/core/AnalyticConstants.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2005-2015 WSO2, Inc. (http://wso2.com)
```

Comment:
This should be : Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.